grafana-plugin/src/models/mattermost/mattermost_channel.ts

```diff
@@ -21,6 +26,28 @@ export class MattermostChannelStore extends BaseStore {
     this.path = '/mattermost/channels/';
   }
 
+  @action.bound
+  async updateMattermostChannels() {
+    const response = await makeRequest(this.path, {});
+
+    const items = response.reduce(
+      (acc: any, mattermostChannel: MattermostChannel) => ({
+        ...acc,
+        [mattermostChannel.id]: mattermostChannel,
+      }),
+      {}
+    );
+
+    runInAction(() => {
+      this.items = {
+        ...this.items,
+        ...items,
+      };
+
+      this.currentTeamToMattermostChannel = response.map((mattermostChannel: MattermostChannel) => mattermostChannel.id);
```

Comment:
The name of this property / observable is misleading imho. What it stores is a list of channel ids from the last request.

But anyway, it seems we're just using it to find out whether there are any channels already fetched, like that:
```
mattermostChannelStore.currentTeamToMattermostChannel?.length > 0;
```

The same can be done by checking if `Object.keys(MattermostChannelStore).length > 0`. So we probably don't need this observable at all?